### PR TITLE
Feature: Add class and table Token

### LINKED ISSUE #138: Add table `Token`
## User stories
As a user I want get my token saved into the database in order to send requests and get recognized.

## Acceptance Criteria
- [x] Add table `Token` without any foreign keys 
- [x] add class to table-list in constants.py

### PR DESCRIPTION
**Pull request by antonykamp**
Fixes #138 

Add table and class Token. Adds tests. Updates database schema in wiki.

## PR checklist

- [x] Acceptance criteria fulfilled
- [x] Additional features are tested
- [x] Docs (code, wiki & diagrams) updated
- [ ] Dev-branch has been merged into local branch to resolve conflicts
- [ ] Tests and linter have passed AFTER local merge
- [ ] Another dev reviewed and approved

### CODE REVIEW

File: src/implementor/models.py [hunk 1]
```diff
@@ -4,6 +4,22 @@
 from src.base_model import BaseModel
 
 
+class Token(BaseModel):
+    """Represents a token."""
+
+    class Schema(BaseModel.Schema):
+        permission = marsh.fields.String(required=True)
```
Comment (SaturnHafen) [inline src/implementor/models.py:11]: We have some kind of permission names and not a bitset indicating the different permissions (why do we need different permission levels?)

File: tests/implementor/test_implementor_models.py [hunk 2]
```diff
@@ -1,8 +1,80 @@
...
+import marshmallow as marsh
+import peewee
 import pytest
 
-from src.implementor.models import Run
+from src.implementor.models import Run, Token
+
+
+class TestImplementorModelsToken:
+    """Test the Token table and the serialization/deserialization."""
```
Comment (SaturnHafen) [inline tests/implementor/test_implementor_models.py]:
```suggestion
    """Test the token table and the serialization/deserialization."""
```

**Review by instant-sky — APPROVED** [review]
Looks good! I like that everything is tested! 🧪

**Review by SaturnHafen — COMMENTED** [review]
Looks good!

File: src/implementor/models.py (hunk 1 above)
Comment (antonykamp) [reply src/implementor/models.py:11]: Good point! I thought, we'll have permission names defined in the implementor component 👍

**Review by SaturnHafen — APPROVED** [review]

**Pull request merged**